# Format:  Position / Orientation 
# Ws_x : [X, Y, Z, qx, qy, qz, qw ] 
possible_poses:
  pose_1: [ 0, 0, 0, 0, 0, 0, 1 ] 
  pose_2: [ 0, 0.24156, 0, 0, 0, 0, 1 ] 
  pose_3: [ 0, 0.37129, 0, 0, 0, 0, 1 ] 
  pose_4: [ 0, 0.4802, 0, 0, 0, 0, 1 ] 
  pose_5: [ 0, 0.59237, 0, 0, 0, 0, 1 ] 
  pose_6: [ 0, 0.68893, 0, 0, 0, 0, 1 ] 
  pose_7: [ 0, 0.77866, 0, 0, 0, 0, 1 ] 
  pose_8: [ 0, 0.9396, 0, 0, 0, 0, 1 ] 
  pose_9: [ 0, 1.0979, 0, 0, 0, 0, 1 ] 
  pose_10: [ 0, 1.2368, 0, 0, 0, 0, 1 ] 
  pose_11: [ 0, -0.00128, 0, 0, 0, 0, 1 ] 
  pose_12: [ 0, -0.14888, 0, 0, 0, 0, 1 ] 
  pose_13: [ 0, -0.28933, 0, 0, 0, 0, 1 ] 
  pose_14: [ 0, -0.35988, 0, 0, 0, 0, 1 ] 
  pose_15: [ 0, -0.43335, 0, 0, 0, 0, 1 ] 
  pose_16: [ 0, -0.54064, 0, 0, 0, 0, 1 ] 
  pose_17: [ 2.6375, -0.04271, 0, 0, 0, 0.99998, -0.00578 ] 
  pose_18: [ 2.6379, -0.16363, 0, 0, 0, 0.99998, -0.00577 ] 
  pose_19: [ 2.6381, -0.27124, 0, 0, 0, 0.99998, -0.00577 ] 
  pose_20: [ 2.6384, -0.37624, 0, 0, 0, 0.99998, -0.00577 ] 
  pose_21: [ 2.6386, -0.43249, 0, 0, 0, 0.99998, -0.00571 ] 
  pose_22: [ 2.6388, -0.51604, 0, 0, 0, 0.99998, -0.00571 ] 
  pose_23: [ 2.639, -0.58138, 0, 0, 0, 0.99998, -0.00572 ] 
  pose_24: [ 2.6392, -0.65843, 0, 0, 0, 0.99998, -0.00567 ] 
  pose_25: [ 2.6395, -0.76052, 0, 0, 0, 0.99998, -0.00553 ] 
  pose_26: [ 2.6397, -0.87983, 0, 0, 0, 0.99998, -0.00553 ] 
  pose_27: [ 2.6011, -0.03132, 0, 0, 0, 0.99993, 0.01146 ] 
  pose_28: [ 2.6027, 0.05905, 0, 0, 0, 0.99993, 0.01165 ] 
  pose_29: [ 2.6047, 0.19459, 0, 0, 0, 0.99993, 0.01164 ] 
  pose_30: [ 2.6067, 0.3376, 0, 0, 0, 0.99993, 0.01163 ] 
  pose_31: [ 2.6087, 0.47834, 0, 0, 0, 0.99993, 0.01162 ] 
  pose_32: [ 2.6097, 0.54302, 0, 0, 0, 0.99993, 0.01162 ] 
  pose_33: [ 2.6112, 0.64801, 0, 0, 0, 0.99993, 0.01161 ] 
  pose_34: [ 2.6128, 0.75917, 0, 0, 0, 0.99993, 0.01161 ] 
  pose_35: [ 2.6137, 0.82092, 0, 0, 0, 0.99993, 0.0116 ] 
  pose_36: [ 2.6152, 0.92623, 0, 0, 0, 0.99993, 0.0116 ] 
  pose_37: [ 2.6164, 1.0111, 0, 0, 0, 0.99993, 0.01159 ] 
  pose_38: [ 2.618, 1.1248, 0, 0, 0, 0.99993, 0.01159 ] 

number_of_poses: 38
